Formula/kong.rb

```diff
@@ -92,3 +94,53 @@ def install
  
  role = traditional
  kic = off
+diff -r -u a/kong/cmd/prepare.lua b/kong/cmd/prepare.lua
+--- a/kong/cmd/prepare.lua	2022-09-12 14:38:55.000000000 +0200
++++ b/kong/cmd/prepare.lua	2022-09-15 10:53:58.000000000 +0200
+@@ -23,8 +23,8 @@
+
+ Example usage:
+  kong migrations up
+- kong prepare -p /usr/local/kong -c kong.conf
+- nginx -p /usr/local/kong -c /usr/local/kong/nginx.conf
++ kong prepare -p HOMEBREW_PREFIX -c kong.conf
++ nginx -p HOMEBREW_PREFIX -c HOMEBREW_PREFIX/nginx.conf
+
+ Options:
+  -c,--conf       (optional string) configuration file
+diff -r -u a/kong/pdk/init.lua b/kong/pdk/init.lua
+--- a/kong/pdk/init.lua	2022-09-12 14:38:55.000000000 +0200
++++ b/kong/pdk/init.lua	2022-09-15 10:54:21.000000000 +0200
+@@ -49,7 +49,7 @@
+ --
+ -- @field kong.configuration
+ -- @usage
+--- print(kong.configuration.prefix) -- "/usr/local/kong"
++-- print(kong.configuration.prefix) -- "HOMEBREW_PREFIX"
+ -- -- this table is read-only; the following throws an error:
+ -- kong.configuration.prefix = "foo"
+
+diff -r -u a/kong/runloop/plugin_servers/process.lua b/kong/runloop/plugin_servers/process.lua
+--- a/kong/runloop/plugin_servers/process.lua	2022-09-12 14:38:55.000000000 +0200
++++ b/kong/runloop/plugin_servers/process.lua	2022-09-15 10:54:10.000000000 +0200
+@@ -61,7 +61,7 @@
+       local env_prefix = "pluginserver_" .. name:gsub("-", "_")
+       _servers[i] = {
+         name = name,
+-        socket = config[env_prefix .. "_socket"] or "/usr/local/kong/" .. name .. ".socket",
++        socket = config[env_prefix .. "_socket"] or "HOMEBREW_PREFIX/" .. name .. ".socket",
```

Comment:
Isn't this code trying to open a socket inside a folder called `HOMEBREW_PREFIX/` (if the config doesn't have a value for `config[env_prefix .. "_socket"]`)?